src/Twilio/TwilioOrgsTokenAuth.cs

```diff
@@ -0,0 +1,161 @@
+using Twilio.Clients;
+using Twilio.Clients.BearerToken;
+using Twilio.Exceptions;
+using Twilio.Http.BearerToken;
+
+namespace Twilio
+{
+    /// <summary>
+    /// Default Twilio Client for bearer token authentication
+    /// </summary>
+    public class TwilioOrgsTokenAuthClient
+    {
+        private static string _accessToken;
+        private static string _region;
+        private static string _edge;
```

Comment:
private static variables can be access outside of class ? If yes, then either these should be only static or private.